src/ft/fc/mod.rs

```diff
@@ -47,6 +48,13 @@ pub fn font_match(config: &ConfigRef, pattern: &PatternRef) -> Option<Pattern> {
     }
 }
 
+/// Reloads the Fontconfig configuration files.
+pub fn bring_config_upto_date() {
+    unsafe {
+        let _ = FcInitBringUptoDate();
+    }
+}
```

Comment:
Seems like you tried to mimic fontconfig's method name here? I personally find it absolutely atrocious tbh.

I'd much rather go with something like `update_config`, like you had it before.